### PR TITLE
New version: Initex.Proxifier version 4.12

### LINKED ISSUE #141680: [Update Request]: Proxifier
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Initex.Proxifier

### Package Version

4.12

### Please describe the changes you would like to see

There is a new release (4.12) available. Download Page Url: https://www.proxifier.com (https://www.proxifier.com/download/ProxifierSetup.exe)

### PR DESCRIPTION
**Pull request by fancybody**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.1.0 :rocket:

- Resolves #141680

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141716)

### CODE REVIEW

File: manifests/i/Initex/Proxifier/4.12/Initex.Proxifier.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Initex.Proxifier
+PackageVersion: '4.12'
+InstallerLocale: en-US
```
Comment (AsciiWolf) [inline manifests/i/Initex/Proxifier/4.12/Initex.Proxifier.installer.yaml:6]:
```suggestion
InstallerLocale: en-US
MinimumOSVersion: 10.0.0.0
```

**Review by stephengillie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by Trenly** [comment]
Trying to debug the issue preventing this from merging

**Comment by Trenly** [comment]
* https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**